src/ORM/Locator/TableLocator.php

```diff
@@ -27,7 +28,7 @@
 /**
  * Provides a default registry/factory for Table objects.
  */
-class TableLocator implements LocatorInterface
+class TableLocator extends AbstractLocator implements LocatorInterface
```

Comment:
The interface is already implemented and should be removed here.

Reply:
This is `ORM\LocatorInterace` not `Datsource\Locator\LocatorInterface`.

Reply:
Hah ok :) My bad.